gradle.properties

```diff
@@ -8,7 +8,7 @@ android.useAndroidX=true
 systemProp.org.gradle.internal.publish.checksums.insecure=true
 
 GROUP=com.squareup.workflow1
-VERSION_NAME=1.10.0-beta06-SNAPSHOT
+VERSION_NAME=1.10.0-beta06-state3-SNAPSHOT
```

Comment:
This looks like an accident.

Reply:
yes, just keeping track of the Snapshot version I'm using for internal canary testing.